### DISCUSSION TITLE
Firmware.bin

### DISCUSSION THREAD

**Discussion by josefkozisek** [jomjol/AI-on-the-edge-device#3157] [Q&A] [answered]
Hi all,
I can find partitions.bin and bootloader.bin here, but from where can be download correct version of firmware.bin?
Thank you

**Comment by SybexX** [comment, marked as answer]
https://jomjol.github.io/AI-on-the-edge-device-docs/Installation/#manual-flashing

for release version:
--------------------
https://github.com/jomjol/AI-on-the-edge-device/releases
You will find all the files you need in the AI-on-the-edge-device__manual-setup__v15.7.0.zip 

for rolling version:
-------------------
https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10028173125
You will find all the files you need in the AI-on-the-edge-device__manual-setup__rolling_(f534741).zip

**Reply by josefkozisek** [reply]
Great, it works well. Thank you.
